app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Indentation like this has better readability, fix it in all similar cases


```suggestion
    def __init__(self,
                comfort_class: int,
                clean_mark: int, 
                brand: str) -> None:
```